### PR TITLE
Add com.github.imagefanreloaded

### PR DESCRIPTION
**Pull request by mihnea-radulescu**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: com.github.imagefanreloaded.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        {
+            "name": "ImageFanReloaded-Avalonia",
+            "buildsystem": "simple",
+            "sources": [
+				{
+					"type": "file",
+					"only-arches": [
+						"x86_64"
+					],
+					"url": "https://github.com/mihnea-radulescu/imagefanreloaded/releases/download/1.2023.08.28/ImageFanReloaded-Avalonia-Linux-x64-Flathub.zip",
```
Comment (hfiguiere) [inline com.github.imagefanreloaded.json:23]: how does this work on the aarch64 flatpak that is built?

File: com.github.imagefanreloaded.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "app-id" : "com.github.imagefanreloaded",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "22.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "ImageFanReloaded-Avalonia",
+    "finish-args" : [
+		"--socket=x11",
+        "--share=ipc",
+        "--device=dri",
+		"--filesystem=host:ro"
```
Comment (hfiguiere) [inline com.github.imagefanreloaded.json:11]: host is a very broad filesystem permission

File: com.github.imagefanreloaded.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+    "app-id" : "com.github.imagefanreloaded",
```
[the remark this answers is not in the document]
Comment (mihnea-radulescu) [reply com.github.imagefanreloaded.json]: Thank you for the feedback, I've fixed app-id and its related data.

File: com.github.imagefanreloaded.json (hunk 2 above)
Comment (mihnea-radulescu) [reply com.github.imagefanreloaded.json:11]: Thank you for the feedback!

As the application is an image viewer, it needs permissions to all folders that potentially contain images. The sensitive system folders are already excluded by flatpak, according to the official documentation, and the access that I need is read-only.

If you think it a better idea to only require read-only access on /home, /media and /mnt, please let me know.

File: com.github.imagefanreloaded.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{
+    "app-id" : "io.github.mihnearadulescu.imagefanreloaded",
```
Comment (hfiguiere) [inline com.github.imagefanreloaded.json:2]: the github username for the repository doesn't match.

File: com.github.imagefanreloaded.json (hunk 1 above)
Comment (mihnea-radulescu) [reply com.github.imagefanreloaded.json:23]: Thank you for raising this, I've duly updated the manifest by adding a separate aarch64 build.

File: com.github.imagefanreloaded.json (hunk 4 above)
Comment (mihnea-radulescu) [reply com.github.imagefanreloaded.json]: Done, thanks.

Comment (hfiguiere) [reply com.github.imagefanreloaded.json]: this what not what I was saying

File: io.github.mihnearadulescu.imagefanreloaded.json [hunk 5]
```diff
@@ -0,0 +1,45 @@
+{
+    "app-id" : "io.github.mihnearadulescu.imagefanreloaded",
```
Comment (hfiguiere) [inline io.github.mihnearadulescu.imagefanreloaded.json:2]:
```suggestion
    "app-id" : "io.github.mihnea_radulescu.imagefanreloaded",
```
This is what the app-id should be.

Comment (mihnea-radulescu) [reply io.github.mihnearadulescu.imagefanreloaded.json]: Updated, thanks!

File: com.github.imagefanreloaded.json (hunk 2 above)
Comment (mihnea-radulescu) [reply com.github.imagefanreloaded.json]: I've decided to narrow down the permissions, seeing that the flatpak application functions well with a restricted permission set.

**Comment by hfiguiere** [comment]
bot, build io.github.mihnea_radulescu.imagefanreloaded

**Pull request closed without merging**